NCEP/cronjob_ursa.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash --login
+
+cd /scratch3/NAGAPE/gpu-ai4wp/Linlin.Cui/eagle_solo
+
+# Get the UTC hour and calculate the time in the format yyyymmddhh
+current_hour=$(date -u +%H)
+current_hour=$((10#$current_hour))
+
+if (( $current_hour >= 0 && $current_hour < 6 )); then
+    datetime=$(date -u -d 'today 00:00')
+elif (( $current_hour >= 6 && $current_hour < 12 )); then
+    datetime=$(date -u -d 'today 06:00')
+elif (( $current_hour >= 12 && $current_hour < 18 )); then
+    datetime=$(date -u -d 'today 12:00')
+else
+    datetime=$(date -u -d 'today 18:00')
+fi
+
+# Calculate time 6 hours before
+#curr_datetime=$(date -u -d "$time" +'%Y%m%d%H')
+curr_datetime=$( date -d "$datetime 6 hour ago" "+%Y%m%d%H" )
+prev_datetime=$( date -d "$datetime 12 hour ago" "+%Y%m%d%H" )
+
+echo "Current state: $curr_datetime"
+echo "6 hours earlier state: $prev_datetime"
+
+echo "Job 1 is running"
+sh /scratch3/NAGAPE/gpu-ai4wp/Linlin.Cui/eagle_solo/gc_prepdata_ursa.sh $curr_datetime $prev_datetime
```

Comment:
Can we replace the local directory to relative to the repo structure (e.g. $PWD/NCEP/gc_prepdata_ursa.sh, $PWD is the location where graphcast code is checked out), so everyone can run the code from the repo.